specification/purview/Azure.Analytics.Purview.DataMap/routes.tsp

```diff
@@ -0,0 +1,2355 @@
+  @route("/atlas/v2/entity/guid/{guid}/classifications")
+  @put
+  updateClassifications is AtlasOperation<
+    {
+      @doc("The globally unique identifier of the entity.")
+      @path
+      guid: string;
+
+      @doc("An array of classifications to be updated.")
+      @body
+      body: AtlasClassification[];
+    },
+    void,
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  #suppress "@azure-tools/typespec-azure-core/operation-missing-api-version" "This is Atlas API, which does not require api version"
+  @doc("""
+Get complete definition of an entity given its type and unique attribute.
+
+In
+addition to the typeName path parameter, attribute key-value pair(s) can be
+provided in the following format:
+attr:\\<attrName>=<attrValue>.
+
+NOTE: The
+attrName and attrValue should be unique across entities, eg.
+qualifiedName.
+
+The REST request would look something like this:
+GET
+/v2/entity/uniqueAttribute/type/aType?attr:aTypeAttribute=someValue.
+""")
+  @route("/atlas/v2/entity/uniqueAttribute/type/{typeName}")
+  @get
+  getByUniqueAttributes is AtlasOperation<
+    {
+      @doc("The name of the type.")
+      @path
+      typeName: string;
+
+      @doc("Whether to return minimal information for referred entities.")
+      @query
+      minExtInfo?: boolean;
+
+      @doc("Whether to ignore relationship attributes.")
+      @query
+      ignoreRelationships?: boolean;
+
+      @doc("""
+The qualified name of the entity. (This is only an example. qualifiedName can
+be changed to other unique attributes)
+""")
+      @query("attr:qualifiedName")
+      attribute?: string;
+    },
+    AtlasEntityWithExtInfo,
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  #suppress "@azure-tools/typespec-azure-core/operation-missing-api-version" "This is Atlas API, which does not require api version"
+  @doc("""
+Update entity partially - Allow a subset of attributes to be updated on an
+entity which is identified by its type and unique attribute eg:
+Referenceable.qualifiedName. Null updates are not possible.
+
+In addition to the
+typeName path parameter, attribute key-value pair(s) can be provided in the
+following format:
+
+attr:<attrName>=<attrValue>.
+NOTE: The attrName and
+attrValue should be unique across entities, eg. qualifiedName.
+
+The REST
+request would look something like this:
+PUT
+/v2/entity/uniqueAttribute/type/aType?attr:aTypeAttribute=someValue.
+""")
+  @route("/atlas/v2/entity/uniqueAttribute/type/{typeName}")
+  @put
+  partialUpdateByUniqueAttributes is AtlasOperation<
+    {
+      @doc("The name of the type.")
+      @path
+      typeName: string;
+
+      @doc("""
+The qualified name of the entity. (This is only an example. qualifiedName can
+be changed to other unique attributes)
+""")
+      @query("attr:qualifiedName")
```

Comment:
This does not look correct - isn't this supposed to be arbitrary key/value pairs? 
```suggestion
      @query("attr:qualifiedName")
```

Reply:
The is the current documented API behavior. https://learn.microsoft.com/en-us/rest/api/purview/datamapdataplane/entity/get-by-unique-attributes?view=rest-purview-datamapdataplane-2023-09-01&tabs=HTTP

Reply:
We have an issue open for this @johanste, I'm looking for it to link. This is currently at the same level of support as the team had with swagger.

Reply:
Couldnt find the issue Johan, going to ping some other tsp folks if they recall what happened with the original issue, otherwise I will open a new one.